src/Sylius/Behat/Context/Api/Admin/ManagingProductReviewsContext.php

```diff
@@ -0,0 +1,274 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Client\ApiPlatformStateMachineClientInterface;
+use Sylius\Behat\Service\SharedStorage;
+use Sylius\Behat\Service\SharedStorageInterface;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Core\ProductReviewTransitions;
+use Sylius\Component\Review\Model\ReviewInterface;
+use Webmozart\Assert\Assert;
+
+final class ManagingProductReviewsContext implements Context
+{
+    /** @var string */
+    private $resource;
+
+    /** @var ApiClientInterface */
+    private $client;
+
+    /** @var ApiPlatformStateMachineClientInterface */
+    private $stateMachineClient;
+
+    /** @var SharedStorage */
+    private $sharedStorage;
+
+    public function __construct(
+        string $resource,
+        ApiClientInterface $client,
+        ApiPlatformStateMachineClientInterface $stateMachineClient,
+        SharedStorageInterface $sharedStorage
+    ) {
+        $this->resource = $resource;
+        $this->client = $client;
+        $this->stateMachineClient = $stateMachineClient;
+        $this->sharedStorage = $sharedStorage;
+    }
+
+    /**
+     * @When I browse product reviews
+     * @When I want to browse product reviews
+     */
+    public function iWantToBrowseProductReviews(): void
+    {
+        $this->client->index($this->resource);
+    }
+
+    /**
+     * @Then I should (also) see the product review :title in the list
+     */
+    public function iShouldSeeTheProductReviewTitleInTheList(string $title): void
+    {
+        Assert::true(
+            $this->isItemOnIndex('title', $title),
+            sprintf('Product review with title %s does not exist', $title)
+        );
+    }
+
+    /**
+     * @Then I should see a single product review in the list
+     * @Then I should see :amount reviews in the list
+     */
+    public function iShouldSeeReviewsInTheList(int $amount = 1): void
+    {
+        Assert::same($this->client->countCollectionItems(), $amount);
+    }
+
+    /**
+     * @When I want to modify the :productReview product review
+     */
+    public function iWantToModifyTheProductReview(ReviewInterface $productReview): void
+    {
+        $this->client->buildUpdateRequest($this->resource, (string) $productReview->getId());
+    }
+
+    /**
+     * @When I change its title to :title
+     */
+    public function iChangeItsTitleTo(?string $title = null): void
+    {
+        if ($title !== null) {
+            $this->client->addRequestData('title', $title);
+        }
+    }
```

Comment:
```suggestion
    /**
     * @When I change its title to :title
     */
    public function iChangeItsTitleTo(string $title): void
    {
        $this->client->addRequestData('title', $title);
    }
```
You don't need set as null because your step doesn't specify as empty